kolibri/plugins/coach/assets/src/modules/pluginModule.js

```diff
@@ -77,11 +77,7 @@ export default {
     handleCoachPageError(store, errorObject) {
       const authErrorCodes = [401, 403, 404, 407];
       logging.error(errorObject);
-      if (
-        errorObject.status &&
-        errorObject.status.code &&
-        authErrorCodes.includes(errorObject.status.code)
-      ) {
+      if (errorObject.response.status && authErrorCodes.includes(errorObject.response.status)) {
         store.dispatch('handleApiError', '');
       } else {
         store.dispatch('handleApiError', errorObject);
```

Comment:
We need to update `AppError.vue` to handle the new shape of `errorObject`

Reply:
Good catch, thanks.